shared/src/store/action-reducers/transfer-point.ts

```diff
@@ -256,12 +256,14 @@ export namespace TransferPointActionReducers {
                     draftState.transferPoints
                 )) {
                     const transferPoint =
-                        draftState.transferPoints[_transferPointId];
+                        draftState.transferPoints[_transferPointId]!;
```

Comment:
```ts
for (const [
    _transferPointId,
    transferPoint,
] of Object.entries(draftState.transferPoints)) {
```

Reply:
I can even use `.values` as the id is only used to get the element.